packages/components/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid-1-2 {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 2rem;
+}
+
+.grid-1-3 {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    gap: 0.625rem;
+    justify-content: center;
+    align-items: center;
+}
+
+.grid-1-4 {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    grid-template-rows: auto;
+    gap: 0.625rem;
+}
```

Comment:
It is better to create a class for common properties in that way you do not have to rewrite the code 
Ex: you could have created a class named grid
. grid {
     width: 100%;
    height: auto;
    display: grid;
}

Reply:
thanks mate

Reply:
using one class name is good right